front/app/containers/Admin/pagesAndMenu/containers/PagesMenu/messages.ts

```diff
@@ -14,4 +14,12 @@ export default defineMessages({
     id: 'app.modules.navbar.admin.containers.createCustomPageButton',
     defaultMessage: 'Create custom page',
   },
+  addProject: {
+    id: 'app.modules.navbar.admin.containers.addProject',
+    defaultMessage: 'Add project to navbar',
+  },
+  navBarMaxItems: {
+    id: 'app.modules.navbar.admin.containers.navBarMaxItemsNumber',
+    defaultMessage: 'You can only add up to 5 items to the navigation bar',
```

Comment:
Nit: I wonder if "5 items" could be confusing if the admin sees 7 items (including home and all projects).

Reply:
I had it at 7 initially then saw the copy from Rob said 5 so I changed it. I guess let's go with this and see if it causes confusion. It's just copy so we can easily adjust it.